drivers/lte_link_control/lte_lc.c

```diff
@@ -153,23 +162,51 @@ static int w_lte_lc_init(void)
 
 static int w_lte_lc_connect(void)
 {
-	if (at_cmd_write(network_mode, NULL, 0, NULL) != 0) {
-		return -EIO;
-	}
+	int err;
+	char *current_network_mode = (char *)nw_mode_preferred;
+	bool use_fallback = false;
 
+	at_cmd_set_notification_handler(at_handler);
 	k_sem_init(&link, 0, 1);
 
-	at_cmd_set_notification_handler(at_handler);
+retry:
+	if (IS_ENABLED(CONFIG_LTE_NETWORK_USE_FALLBACK) &&
+	    (use_fallback != false)) {
```

Comment:
`(use_fallback != false)` looks awkward to me. `use_fallback` is a boolean variable, can't we simply:
```
if (IS_ENABLED(CONFIG_LTE_NETWORK_USE_FALLBACK) && use_fallback) {
```

Reply:
Yeah, this looks weird.. Fixed.